code/game/atoms.dm

```diff
@@ -30,6 +30,12 @@
 
 	if(color)
 		add_atom_colour(color, FIXED_COLOUR_PRIORITY)
+
+	if(!atom_colours)
+		atom_colours = list(ADMIN_COLOUR_PRIORITY = null, \
+		TEMPORARY_COLOUR_PRIORITY = null, \
+		WASHABLE_COLOUR_PRIORITY = null, \
+		FIXED_COLOUR_PRIORITY = null)

```

Comment:
no, just initialize it in structure/spacevine. The whole atom colour code is supposed to work when the list isn't initialized, the error is in the spacevine code.

The best solution would be to grayscale the spacevines icon and give the default vine its green hue by adding `add_atom_colour("#greenhuegoeshere", FIXED_COLOUR_PRIORITY)` in structure/spacevine/New().